app/lib/presentation/organisms/practice_collection_organism.dart

```diff
@@ -30,6 +33,20 @@ class _PracticeCollectionOrganismState
     currentCard = cards[currentCardIndex];
   }
 
+  Future<void> _speakCardContent() async {
+    if (currentCard == null) return;
+
+    String textToSpeak = "";
+
+    if (currentCard!.answer?.isNotEmpty == true) {
+      textToSpeak += currentCard!.answer!;
+    }
+
+    if (textToSpeak.isNotEmpty) {
+      await _ttsController.speak(textToSpeak);
+    }
+  }
+
```

Comment:
This method is redundant.
currentCard will never be null when we call it.
Just call it like this  
` TtsController.instance.speak(currentCard!.answer??'');`